pyccel/commands/pyccel_clean.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+""" Module containing scripts to remove pyccel generated objects
+"""
+import os
+import shutil
+import sysconfig
+
+ext_suffix = sysconfig.get_config_var('EXT_SUFFIX')
+
+def pyccel_clean(path_dir = None, recursive = True, remove_shared_libs = False):
+    """ Remove __pyccel__ and __epyccel__ folders as well
+    as any python shared libraries from the directory path_dir
+
+    Parameters
+    ----------
+    path_dir  : str
+                The path to the folder which should be cleaned
+                Default : current working directory
+    recursive : bool
+                Indicates whether the function should recurse
+                into sub-folders
+                Default : True
+    """
```

Comment:
Please update the docstring with the parameter `remove_shared_libs`.

Reply:
Fixed 832526a